docs/tutorial/snapcraft-guide.md

```diff
@@ -0,0 +1,181 @@
+# Snapcraft Guide (Ubuntu Software Center & More)
+
+This guide provides information on how to package your Electron application
+for the Ubuntu Software Center or any other Snapcraft environment.
+
+## Background and Requirements
+
+Together with the broader Linux community, Canonical aims to fix many of the
+common software installation problems with the [`snapcraft`](snapcraft.io)
+project. Snaps are containerized software packages that include required
+dependencies, auto-update, and work on all major Linux distributions without
+system modification.
+
+Snapcraft is the primary way to get your application into the Ubuntu Software
+Center, but the underlying [Snap Store](snapcraft-store) supports all major
+Linux distributions, too.
+
+There are two ways to create a `.snap` file:
+
+1) Using `electron-installer-snap`, which takes `electron-packager's` output
+2) Using an already created `.deb` package
+3) Using [`electron-forge`](electron-forge) or
+   [`electron-builder`](electron-builder), both tools that come with `snap`
+   support out of the box (not further documented here, please see those
+   frameworks for further guidance)
+
+In both cases, you will need to have the `snapcraft` tool installed. We
+recommend building on Ubuntu 16.04 (or the current LTS).
+
+```sh
+snap install snapcraft --classic
+```
+
+While it _is possible_ to install `snapcraft` on macOS using Homebrew, you are
+less likely to encounter issues when running `snapcraft` on an actual Linux
+distribution.
```

Comment:
You can install `snapcraft` on macOS using Homebrew, but this version does not yet build snaps. It is however useful for managing releases in the Snap Store.